apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.spec.ts

```diff
@@ -226,4 +228,86 @@ describe('transformCase', () => {
       expect(res.isStatementDeadlineExpired).toBe(false)
     })
   })
+
+  describe('appealInfo', () => {
+    it('should be undefined when no court end time is set', () => {
+      // Arrange
+      const theCase = {} as Case
+
+      // Act
+      const res = transformCase(theCase)
+
+      // Assert
+
+      expect(res.appealDeadline).toBeUndefined()
+      expect(res.appealedByRole).toBeUndefined()
+      expect(res.appealedDate).toBeUndefined()
+      expect(res.hasBeenAppealed).toBe(false)
+      expect(res.canBeAppealed).toBe(false)
+    })
+
+    it('should have appeal deadline and hasBeenAppealed set to false', () => {
```

Comment:
Please consider adding a more specific description here, i.e. "should have appeal deadline and hasBeenAppealed set to false **when something** "